Dockerfile

```diff
@@ -74,3 +74,9 @@ RUN apt-get update -y \
 	make install && \
 	apt-get autoclean -y && apt-get --purge --yes autoremove && \
 	rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```

Comment:
should these be removed, so there's no need to run `apt-get update -y` a second time? 

otherwise, looks good!

Reply:
Yeah, the apt-get calls should be consolidated - I was just trying to minimize the number of changed layers to speed development.